docs/doxygen/Doxyfile.in

```diff
@@ -736,8 +736,8 @@ EXCLUDE                = @abs_top_builddir@/cmake/msvc \
                          @abs_top_builddir@/grc \
                          @abs_top_builddir@/_CPack_Packages \
                          @abs_top_srcdir@/cmake \
-                         @abs_top_srcdir@/gr-utils/python/modtool/gr-newmod \
-                         @abs_top_builddir@/gr-utils/python/modtool/gr-newmod \
+                         @abs_top_srcdir@/gr-utils/modtool/templates/gr-newmod/
```

Comment:
Works good, but wouldn't be prettier if the line ended with \ like all the others?

Reply:
Errr.... yes.